datafusion/sql/src/unparser/plan.rs

```diff
@@ -352,6 +352,20 @@ impl Unparser<'_> {
                 )
             }
             LogicalPlan::Sort(sort) => {
+                if let Some(fetch) = sort.fetch {
+                    let new_sort = Arc::new(LogicalPlan::Sort(Sort {
+                        expr: sort.expr.clone(),
+                        input: Arc::clone(&sort.input),
+                        fetch: None,
+                    }));
+                    let limit = Arc::new(LogicalPlan::Limit(Limit {
```

Comment:
Is it possible to unparse without creating intermediate (fake) plan nodes?

Reply:
Thanks @findepi. Good idea. I tried it in d98169f. It's much better.

Reply:
thanks!